### DISCUSSION TITLE
mission to Saeus

### DISCUSSION THREAD

**Discussion by SamBloggins** [endless-sky/endless-sky#7098] [Q&A] [answered]
How do I get there?  I have the expansion extensions, yet the place does not seem to exist.  Help please

**Comment by mOctave** [comment, marked as answer]
I don't believe there is a place called Saeus in the vanilla game. It appears to be added by the Enclave plugin.
By looking at the map files, it appears that you have to go through a system called "Five" to get there. I'm not sure where the wormhole to Enclave space is (if applicable) or if you have to use a jump drive, since I don't play with that plugin.

Copying the map.txt file off the internet and opening it in Endless Sky's map editor, I was able to get the following image:
<img width="1133" alt="Map of Enclave" src="https://user-images.githubusercontent.com/73318970/183266977-a2777c8d-400b-4df3-8329-ccb16cf0fd5f.png">
Saeus is hiding between Arkhos, Cymexes, Five, and Theurrhaeus.

**Comment by SamBloggins** [comment]
Thank you for this I will try to find this map Sent from Mail for Windows From: mOctaveSent: August 6, 2022 17:46To: endless-sky/endless-skyCc: SamBloggins; AuthorSubject: Re: [endless-sky/endless-sky] mission to Saeus (Discussion #7098) I don't believe there is a place called Saeus in the vanilla game. It appears to be added by the Enclave plugin.By looking at the map files, it appears that you have to go through a system called "Five" to get there. I'm not sure where the wormhole to Enclave space is (if applicable) or if you have to use a jump drive, since I don't play with that plugin.Copying the map.txt file off the internet and opening it in Endless Sky's map editor, I was able to get the following image:Saeus is hiding between Arkhos, Cymexes, Five, and Theurrhaeus.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
